package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ng-packagr",
-  "version": "2.0.0-rc.9",
+  "version": "2.0.0-rc.10",
```

Comment:
Hi @ahmedGomaa90 , pull requests must not update the version numbers. Can you submit the PR with source code changes only?